rewrite-gradle/src/main/resources/META-INF/rewrite/gradle.yml

```
@@ -44,6 +44,13 @@ recipeList:
       filePattern: gradle.properties
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.gradle.GradleBestPractices
+displayName: Apply Gradle Best Practices
+description: Applies a curated set of best practices for Gradle projects.
+recipeList:
+  - org.openrewrite.gradle.ChangeTaskToTasksRegister
+---
```

Might make sense to tie in the Gradle upgrades, to have this match what we do with other best practices recipes.
```suggestion
recipeList:
  - org.openrewrite.gradle.MigrateToGradle8
  - org.openrewrite.gradle.ChangeTaskToTasksRegister
---
```